### ISSUE TITLE
[Accessibility] : In DetailList  Focus is going to non-interactive column headers 

### ISSUE THREAD

**Issue by v-sagsah** [microsoft/fluentui#13068]
**Browser Details: ​**
OS: Windows 10 (Version 1903, Build: 18362.175)
Chromium Edge 81.0.416.41 (Official build) beta (64-bit)

**Issue :**
In DetailsList control, if the column header does not have any onclick() action to perform, in that case the focus is going to to non-interactive column headers .

Here is the repo code of the issue
https://codepen.io/sagsah/full/oNjqdGP

**Expected Result: ​**
As the column headers in the detailList are non-interactive, Keyboard focus should not go to them

**User Impact:** ​
Keyboard users will get confused if focus is going to the non-interactive column headers, but user cannot perform any operation.

**Comment by aneeshack4** [comment]
@v-sagsah This is incorrect - keyboard focus must go on the column headers so that ATs (screenreaders like Narrator) can read out the text inside each column header.

**Comment by v-sagsah** [comment]
Thank you @aneeshack4
